.github/workflows/tests.yml

```
@@ -0,0 +1,56 @@
+name: Tests
+
+on:
+  pull_request:
+
+jobs:
+  phpunit:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        php: [8.1, 8.0, 7.4]
+        wp_version: ["latest"]
+        can_fail: [false]
+        multisite: [0,1]
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ALLOW_EMPTY_PASSWORD: yes
+        ports:
+          - 3306:3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    continue-on-error: ${{ matrix.can_fail }}
+
+    name: WordPress ${{ matrix.wp_version }} @ PHP ${{ matrix.php }} (WP_MULTISITE=${{ matrix.multisite }})
```

If you make `matrix.wp_version` an environment variable, this will need updating to something like the following:

```suggestion
    name: "WordPress $WP_VERSION @ PHP ${{ matrix.php }} (WP_MULTISITE=${{ matrix.multisite }})"
```